code/game/objects/items/plushes.dm

```diff
@@ -473,18 +473,25 @@
 	icon_state = "narplush"
 	divine = TRUE
 	var/clashing
-	var/is_invoker = TRUE
+	var/invoker_charges = 2
 	gender = FEMALE	//it's canon if the toy is
 
+/obj/item/toy/plush/narplush/examine(mob/user)
+	. = ..()
+	if(invoker_charges == 0)
+		. += "<span class='notice'>It looks tired.</span>"
```

Comment:
`gender = FEMALE //it's canon if the toy is`
```suggestion
		. += "<span class='notice'>She looks tired.</span>"
```


Since we are cleaning the plushie code up, perhaps make it more obvious to a cultist that this is an avatar of Nar'sie? It would feel more organic, like the doll is another cult tool, rather than being a joke cheese item.

Perhaps this Plushie is a manifestation of Nar'sies power. To a crewmember its a silly doll, but to a cultist, it is a divine artifact to which they can use to carry out her will.

Reply:
Not really sure how we're going to do that but it is not very common to get (requires hacking the chaplain vendor to get it roundstart)